plugins/org.eclipse.elk.graph.text/src/org/eclipse/elk/graph/text/ElkGraph.xtext

```diff
@@ -22,6 +22,12 @@ ElkNode returns ElkNode:
     'node' identifier=ID ('{'
         ShapeLayout?
         properties+=Property*
+        ('individualSpacing' '{'
```

Comment:
I'd rather write `[ ... ]` because `{ ... }` is used for _elements_ (nodes, labels etc.) in our grammar.

Reply:
It would also be in line with the `layout` element, thus probably more consistent. 

Although, to be honest, I don't see any strong reason to have two different block markers. Yet, the layout one is already there, I guess I'll follow your proposal.